src/ApplyOfflineMap.cpp

```diff
@@ -282,6 +311,42 @@ try {
 		AnnounceStartBlock("Applying first offline map to data");
 	}
 */
+	
+		DataArray3D<int> dataGLLNodesIn;
+		DataArray3D<double> dataGLLJacobianIn;
+		
+		DataArray3D<int> dataGLLNodesOut;
+		DataArray3D<double> dataGLLJacobianOut;
+		Mesh OverlapMesh;
+		Mesh SourceMesh;
+			
+		//FV or FE input mesh
+		if(strInputMesh != ""){
+			Mesh meshSource(strInputMesh);
+			SourceMesh = meshSource;
+			SourceMesh.ConstructReverseNodeArray();
+			SourceMesh.ConstructEdgeMap();
+			//SourceMesh.RemoveZeroEdges();
+			if(fGLL){
+				double dTotalAreaInput = meshSource.CalculateFaceAreas(fContainsConcaveFaces);
+				double dNumericalAreaIn = GenerateMetaData(meshSource,nPin,false,dataGLLNodesIn,dataGLLJacobianIn);
+			}
+		}
+	
+		//FE output mesh
+		if(strOutputMesh != ""){
+			Mesh meshTarget(strOutputMesh);
+			meshTarget.RemoveZeroEdges();
+			double dTotalAreaOutput = meshTarget.CalculateFaceAreas(fContainsConcaveFaces);
+			double dNumericalAreaOut = GenerateMetaData(meshTarget,nPout,false,dataGLLNodesOut,dataGLLJacobianOut);
```

Comment:
Shouldn't there be a block here for `fGLL`, similar to the SourceMesh case ?

Reply:
We actually don't need a block in this case.  The source mesh is always needed regardless of whether it is finite element or finite volume.    The target mesh is not needed when it is a finite volume mesh.